app/src/main/java/com/example/taskyy/ui/screens/AgendaScreen.kt

```diff
@@ -499,10 +503,18 @@ fun AgendaDisplayItem(
     val date = LocalDateTime.ofInstant(instant, ZoneId.systemDefault())
     val formattedDate = DateTimeFormatter.ofPattern("MMMM dd, HH:mm").format(date)
     val color: Color =
-        if (agendaEventItem.eventType == AgendaItemType.REMINDER_ITEM) {
-            Color(android.graphics.Color.parseColor("#f2f6ff"))
-        } else {
-            Color(android.graphics.Color.parseColor("#FFFFFF"))
+        when (agendaEventItem.eventType) {
+            AgendaItemType.REMINDER_ITEM -> {
+                Color(android.graphics.Color.parseColor("#f2f6ff"))
+            }
+
+            AgendaItemType.TASK_ITEM -> {
+                Color(android.graphics.Color.parseColor("#289c74"))
+            }
+
+            else -> {
+                Color(android.graphics.Color.parseColor("#FFFFFF"))
```

Comment:
You don't need to do this with the old Android colors, you can encode the color code as a Hex Long like this: `Color(0xFF289c74)`. The `0x` indicates the number is in hex format, the first `FF` denotes the alpha value and everything that follows after is the color code.